src/app/components/header/header.component.css

```diff
@@ -216,6 +216,7 @@ i {
     display: none;
     background: #009cff;
     box-shadow: none;
+    min-width: 110px;
```

Comment:
This `min-width` is in the max width 992px media screen. This issue is about the desktop version of the submenu items not wide enough. And also the min-width does nothing in the drawer. Please fix it